Mathlib/CategoryTheory/Sites/Monoidal.lean

```diff
@@ -152,19 +152,61 @@ noncomputable def monoidalCategory [(J.W (A := A)).IsMonoidal] [HasWeakSheafify
   inferInstanceAs (MonoidalCategory
     (LocalizedMonoidal (L := presheafToSheaf J A) (W := J.W) (Iso.refl _)))
 
+/-- The monoidal category structure on `Sheaf J A` obtained in `Sheaf.monoidalCategory` is
+braided when `A` is braided. -/
+noncomputable def braidedCategory [(J.W (A := A)).IsMonoidal] [HasWeakSheafify J A]
```

Comment:
Is not it possible to declare `monoidalCategory` as a local instance here, so that we way avoid the multiple `letI` in the declarations?